jobs/involuntary-dissolutions/requirements.txt

```diff
@@ -30,6 +30,9 @@ six==1.15.0
 urllib3==1.26.11
 requests==2.25.1
 cachelib==0.1.1
-git+https://github.com/bcgov/lear.git#egg=legal_api&subdirectory=legal-api
+# install updated legal-api from feature branch for development, 
+# TODO - change it back when the db versioning feature branch merged to main
+git+https://github.com/bcgov/lear.git@feature-db-versioning#egg=legal_api&subdirectory=legal-api
+# git+https://github.com/bcgov/lear.git#egg=legal_api&subdirectory=legal-api
```

Comment:
Now, we'll pull legal-api's update from the feature-db-versioning branch, and switch back to the main later